src/main/java/cloud/orbit/actors/extensions/metrics/dropwizard/OrbitActorExtension.java

```diff
@@ -61,19 +58,35 @@
 
     public static final String ACTOR_METRICS_PIPELINE_NAME = "actor-metrics-pipeline";
 
-    //Timer, per actor instance
-    Map<String, Timer.Context> actorActivationTimers = new ConcurrentHashMap<>();
-    Map<String, Timer.Context> actorLifeSpanTimers = new ConcurrentHashMap<>();
-    Map<String, Timer.Context> actorDeactivationTimers = new ConcurrentHashMap<>();
+    private Map<String, Timer.Context> actorActivationTimers = new ConcurrentHashMap<>();
+    private Map<String, Timer.Context> actorLifeSpanTimers = new ConcurrentHashMap<>();
+    private Map<String, Timer.Context> actorDeactivationTimers = new ConcurrentHashMap<>();
+
+    private MetricRegistry metricRegistry;
 
     public OrbitActorExtension()
```

Comment:
heh... I was about to do that on my latest amend to my commit but decided to hold off as that usually makes the diff not work that great if it isn't picked up properly. Let me try how that will look and can revert the commit in the PR if it doesn't make it convenient to diff.